File: BOJ/CLASS 1/2475.js
// 검증수

const fs = require('fs');
const input = fs
  .readFileSync('예제.txt')
  .toString()
  .split(' ')
  .map((x) => Number(x));

function solution(arr) {
  let answer;

  const newArr = arr.map((x) => x ** 2);
  const sum = newArr.reduce((a, b) => a + b);
  answer = sum % 10;

  return answer;
}

console.log(solution(input));

File: BOJ/CLASS 1/Python/2475.py
# 검증수
data = list(map(int, input().split()))

"""
# 내 풀이 방법
total = 0

for i in data:
    total += i ** 2

print(total % 10)
"""

# 리스트 컴프리헨션 사용
print(sum(i ** 2 for i in data) % 10)